sympy/core/tests/test_numbers.py

```diff
@@ -1749,3 +1749,41 @@ def test_Integer_precision():
     assert Float('1.0', precision=Integer(15))._prec == 15
     assert type(Float('1.0', precision=Integer(15))._prec) == int
     assert sympify(srepr(Float('1.0', precision=15))) == Float('1.0', precision=15)
+
+def test_numpy():
+    from sympy.utilities.pytest import skip
+    import numpy as np
```

Comment:
You have to do `np = import_module('numpy')`, otherwise it raises `ImportError`.

Reply:
Thank you - have changed the imports. Regarding Travis - does it meant I have to make sure local dependencies (and their versions) are the same as those used in the automatic tests?

Reply:
@Trave11er I meant that it's their problem(s) and not caused by your commits.